main.go

```
@@ -314,6 +314,32 @@ func main() {
 				setEditParametersPlain(&cui, &req)
 				app.SetFocus(cui.Request)
 			}
+		} else if event.Key() == tcell.KeyCtrlE {
+			if focus == cui.RequestBody {
+				editor := os.Getenv("EDITOR")
+				currText := cui.RequestBody.GetText()
+
+				_, currentOption := cui.RequestKindDropdown.GetCurrentOption()
+
+				var editedText string
+
+				if app.Suspend(func() {
+					d, errEditor := EditRequestInEditor(currentOption, currText, editor)
+					if errEditor != nil {
+						return
```

I would prefer that this panic for now, once the updated error handling is in place we can let the user know what happened but as-is it will just silently fail which isn't a great user experience